### PR TITLE
Fix a bug, add comments.

### PR DESCRIPTION
**Pull request by wlaun**
- StelScriptMgr: fix bug in Vec3d constructor
- StelMovementMgr::moveToObject moved from public slots to public.
- StelMovementMgr: edited comments for moveToJ2000 and moveToAltAzi,
  explaining how Vec3d works for each mount mode.
- StelMovementMgr: edited comments for turnRight/left/Up/Down and zoomIn/Out

### Description

Fixes # (issue)

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update

### How Has This Been Tested?

**Test Configuration**:
* Operating system: <Name, version number>
* Graphics Card: <Manufacturer (likely Intel, NVidia, AMD?), Model (HD, Geforce, Radeon..., with model number), driver version?>

## Checklist:

- [ ] My code follows the [code style](http://stellarium.org/doc/head/codingStyle.html) of this project.
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/core/StelMovementMgr.hpp [hunk 1]
```diff
@@ -205,8 +207,12 @@ public slots:
 	//! StelMovementMgr* mvmgr = GETSTELMODULE(StelMovementMgr);
 	//! mvmgr->moveToJ2000(pos, mvmgr->mountFrameToJ2000(Vec3d(0., 0., 1.)), mvmgr->getAutoMoveDuration());
 	//! @endcode
+    //! @note core::moveToRaDecJ2000 provides a simpler signature for the same function.
+    //! @note Objects of class Vec3d are 3-dimensional vectors in a rectangular coordinate system. For
+	//!       J2000 positions, the x-axis points to 0h,0°, the y-axis to 6h,0° and the z-axis points to the
+	//!       celestial equator. You may use a constructor defining three components (x,y,z) or the
```
Comment (gzotti) [inline src/core/StelMovementMgr.hpp:213]: z points to the cel. Pole!

File: src/core/StelMovementMgr.hpp [hunk 2]
```diff
@@ -234,13 +247,15 @@ public slots:
 	//! Set the initial Field Of View in degree.
 	void setInitFov(double fov);
 
-	//! Return the inital viewing direction in altazimuthal coordinates
+	//! Return the inital viewing direction in altazimuthal coordinates.
```
Comment (gzotti) [inline src/core/StelMovementMgr.hpp:250]: While we are at it: "initial".

File: src/core/StelMovementMgr.hpp (hunk 1 above)
Comment (wlaun) [reply src/core/StelMovementMgr.hpp]: Fixed, also the inital.

**Comment by wlaun** [comment]
Thanks - the corrections have been made.
-WL

**Pull request merged**